batchflow/research/research.py

```diff
@@ -203,6 +209,7 @@ def get_metrics(self, pipeline, metrics_var, metrics_name, *args,
             include execution information to log file or not
         """
         name = pipeline + '_' + metrics_var
+        name += str(len([item for item in self.executables if item.startswith(name)]))
```

Comment:
This primitive is used multiple times. I'd make a small function smth like
```
def count_startswith(seq, name): return sum(1 for item in seq if item.startswith(name))
```
or a method with `self.executables` instead of `seq`